modules/ob-defining-the-output.adoc

```diff
@@ -0,0 +1,94 @@
+// This module is included in the following assembly:
+//
+// shipwright-builds/configuring-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="ob-defining-the-output_{context}"]
+= Defining the output
+
+In your `Build` CR, you can specify an output location to push the image. When using an external private registry as your output location, you must specify a secret to access the image. You can also specify the annotations and labels for the output image. 
+
+[NOTE]
+====
+When you specify annotations or labels, the output image will get pushed twice. The first push comes from the build strategy and the second push changes the image configuration to add the annotations and labels. 
+====
+
+The following example defines a public registry where the image will be pushed:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: s2i-nodejs-build
+spec:
+  source:
+    git:
+      url: https://github.com/shipwright-io/sample-nodejs
+      contextDir: source-build/
```

Comment:
```suggestion
    contextDir: source-build/
```
@shipsing the contextDir option falls within spec.source . So, the indentation should be like:

~~~
...
spec:
  source:
    git:
      url: <value> 
    contextDir: <value>
~~~

We might need to update this for all the similar examples (using contextDir).